README.md

```diff
@@ -7,22 +7,65 @@
 # pyscaffoldext-markdown
```

Comment:
The Badge for the Docs status is missing after `Build Status`.

Reply:
Thanks Florian

:P let me add it!